sstables/open_info.hh

```diff
@@ -36,7 +36,7 @@ struct entry_descriptor {
         : sstdir(sstdir), ks(ks), cf(cf), generation(generation), version(version), format(format), component(component) {}
 };
 
-entry_descriptor parse_path(const std::filesystem::path& sst_path);
+std::tuple<entry_descriptor, sstring, sstring> parse_path(const std::filesystem::path& sst_path);
```

Comment:
Better to use `struct`, more human-readable that way.

Reply:
The plan is to make the parse_path() return several distinct values. Since only two callers need to get ks.cf and one of them already uses structured bindings, tuple looks the best fit here. I'll patch the second one to use bindings as well.

I first thought to add two out-arguments, but it was uglier. Using struct results in more LOCs for no gain (as per above), so I'd not use it either